src/components/pages/Navbar/Navbar.js

```diff
@@ -20,6 +21,7 @@ const Navbar = ({ isAuthenticated, userProfile, getProfile }) => {
   const [profilePic] = useState('https://joeschmoe.io/api/v1/random');
   const [user, setUser] = useState({});
   const [modal, setModal] = useState(false);
+  const [toggleStatus, setToggleStatus] = useState(false);
```

Comment:
The useState here could also be left to default at an empty value since you're assigning it on the page load with useEffect.